internal/scaffold/role.go

```diff
@@ -143,6 +146,112 @@ func UpdateRoleForResource(r *Resource, absProjectPath string) error {
 	}
 }
 
+// MergeHelmRoleForResource merges incoming new API resource rules with existing deploy/role.yaml
+func MergeHelmRoleForResource(r *Resource, absProjectPath string, roleScaffold Role) error {
```

Comment:
Is this specific to Helm? This can probably be renamed to `MergeRoleForResource`, right?

If it is specfic to Helm, we should move it to somewhere in `internal/scaffold/helm`.